### DISCUSSION TITLE
2.4.7 coming soon with these features

### DISCUSSION THREAD

**Discussion by edmundreinhardt** [IBM/ibmi-tobi#189] [Announcements]
Improved project structure and automate release process.  See the documentation
at https://ibm.github.io/ibmi-bob/#/contributing/getting-started for how to setup
development and run linter and tests and also release using `nox`.  

It is now straightforward to get all the way from making a tweak to getting it published.  
Bringing DevOps to this project!

Thank you @TongkunZhang

**Comment by beckhamrryyaann** [comment]
Great job all!

**Comment by edmundreinhardt** [comment]
FYI 2.4.8 was released Jan 25 and it included a pull request from Susan Bestgen with support for compiling C++ modules.
She also added the ability to configure the ALLOW parameter on CRTCMD.
Thank you Susan @sbestgen 

The bug causing `makei init` to fail was fixed.
